p2p/address_utils.go

```diff
@@ -44,3 +44,15 @@ func stringsToPeerInfos(peers []string) ([]peer.AddrInfo, error) {
 	}
 	return pinfos, nil
 }
+
+// Converts an array of Peer.ID into a string array
+func PeerIdToStringArray(peers []peer.ID) []string {
```

Comment:
comment needs to start with the name of the function for doc generation`// PeerIdToStringArray converts an array of Peer.ID into a string array`